Automattic/newspack-plugin
# fix: update support link and hide gam setup for manager connected sites

### All Submissions:






### Changes proposed in this Pull Request:



See [this thread in slack](https://a8c.slack.com/archives/C015W6BES8J/p1710190649198039?thread_ts=1710190498.816149&cid=C015W6BES8J).

This PR makes some small adjustments to the GAM setup in Ads wizard:

1. Renders the correct onboarding modal depending on connection to manager. When not connected:
![Screenshot 2024-03-14 at 16 49 57](https://github.com/Automattic/newspack-plugin/assets/17905991/09c19f4e-0421-4d6c-8b73-1f96d150466c)

OR when connected:
![Screenshot 2024-03-14 at 16 50 24](https://github.com/Automattic/newspack-plugin/assets/17905991/1359a903-0a46-4950-bf1c-07b09204e6b5)

2. Updates the support link for service account setup
![Screenshot 2024-03-11 at 18 12 46](https://github.com/Automattic/newspack-plugin/assets/17905991/8b990b73-bd8f-46b5-bbca-1f5090d38753)

### How to test the changes in this Pull Request:

1. Go to the Newspack Ads Wizard on a site connected to a network manager site (Newspack > Advertising)
2. Initiate the setup by selecting the `Click here to connect your account` link (Your site will need to be disconnected from GAM for this to appear)
3. On `trunk` you should see the service account upload onboarding flow. On this branch you should see the OAuth connection flow.
4. Now disconnect your site from both GAM and network manager by going to Newspack > Connections and disconnecting Google, then deactivating the manager plugin.
5. Repeat steps 1-3, and this time confirm you see the service account upload flow on this branch (should also still be present on trunk)
6. In the Service account upload onboarding modal, look for the `How to get a service account user for API access` support link and click it. On `trunk` it should take you here: https://support.google.com/admanager/answer/6078734. On this branch it should take you here: https://developers.google.com/ad-manager/api/authentication

### Other information:
